modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHomePage.testcase

```diff
@@ -65,6 +65,24 @@ definition {
 		}
 	}
 
+	@description = "LPS-172661 - Verify clicking on News title will open a modal to display content."
+	@priority = 3
+	test CanSeeWholeNews {
+		task ("Given a user navigates to the EVP home page") {
+			EVPUtils.gotoEVP();
+		}
```

Comment:
Hey @epark18 , I think here you could login as a employee to assert that employee user can see all the news content. You can add this code: `User.logoutAndLoginPG( userLoginEmailAddress = "employee@liferay.com", userLoginFullName = "Employee User");` before `EVPUtils.gotoEVP();` So you can also change the task to:`"Given a Employee user navigates to the EVP home page"`

Reply:
Thank you @MGabriellaS I made the change.